pkgs/top-level/all-packages.nix

```
@@ -36446,6 +36446,10 @@ with pkgs;
 
   go-exploitdb = callPackage ../tools/security/go-exploitdb { };
 
+  gridcoin-research = libsForQt5.callPackage ../applications/blockchains/gridcoin-research {
+    boost = boost17x;
```

```suggestion
    boost = boost179;
```

`boost17x = throw "boost17x has been deprecated in favor of the latest version"; # Added 2023-07-13` 
https://github.com/nixos/nixpkgs/commit/315d1001133a8bd8d0f998a497f8bc4b8eb15012

---

Thanks, I missed that depreciation!
Committed suggested change